build/configs/imxrt1050-evk/loadable_elf_apps/defconfig

```diff
@@ -900,7 +904,6 @@ CONFIG_BUILTIN_APPS=y
 # Examples
 #
 # CONFIG_EXAMPLES_AWS is not set
-CONFIG_EXAMPLES_BINARY_UPDATE=y
```

Comment:
Any reason why you removed binary update?

Reply:
The new defconfig was created with .config content after make menuconfig command. 
This will be the proper setting for defconfig